jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon

```diff
@@ -158,6 +162,10 @@ function galleon_reset_mvn_env() {
     if [ -n "$ORIGINAL_MAVEN_ARGS" ]; then
       MAVEN_ARGS=$ORIGINAL_MAVEN_ARGS
     fi
+
+    if [ -n "$ORIGINAL_MAVEN_ARGS_APPEND" ]; then
+      MAVEN_ARGS_APPEND=$ORIGINAL_MAVEN_ARGS_APPEND
```

Comment:
same comment as above

Reply:
Thank-you, fixed for both MAVEN_ARGS and MAVEN_ARGS_APPEND